.python-version

```diff
@@ -0,0 +1 @@
+3.6.15
```

Comment:
I don't know what this file is, should it be ignored? Python3.6 is EOL...

Reply:
I added this because this is a python version running inside the docker container, I had to learn the python version from there to set up a local environment.

I kept it here so other people can learn the python version quickly.

Reply:
thanks

Reply:
Removed this and gitignored.